app/views/licenses/_license.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="widget">
+  <div class="widget--body">
+    <h3>
+      <%= license.name %>
+      <% if license.default? %>
+        <span class="badge is-tag is-filled">default</span>
+      <% end %>
+      <% unless license.enabled? %>
+        <span class="badge is-tag is-yellow is-filled">
```

Comment:
I don't think this tag (disabled) should be filled. I think it looks better with "unfilled" styles:

![show how this page would look with unfilled badge](https://user-images.githubusercontent.com/21335202/80919765-21b95580-8d6c-11ea-8dc0-f4ffbe8fe5e2.png)